### PR TITLE
updated default value description

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by JuliaMongo — APPROVED** [review]
Hurray!

File: flags.go [hunk 1]
```diff
@@ -76,9 +76,9 @@ func FlagUsages(f *pflag.FlagSet) string {
 		line += "\n    - " + usage
 		if !defaultIsZeroValue(flag) {
 			if flag.Value.Type() == stringType {
-				line += fmt.Sprintf(" (default %q)", flag.DefValue)
+				line += fmt.Sprintf(" This value defaults to %q", flag.DefValue)
```
Comment (gssbzn) [inline flags.go:79]: @JuliaMongo should this be 
```suggestion
				line += fmt.Sprintf(". This option defaults to %q", flag.DefValue)
```

to make it two sentences? also optional, changing value for option

Comment (andreaangiolillo) [reply flags.go:79]: all the flag descriptions end with a full stop https://github.com/mongodb/mongocli/blob/master/internal/usage/usage.go#L18

Comment (andreaangiolillo) [reply flags.go:79]: If we do this, we will end with some usage with `.` and some without it

**Review by gssbzn — APPROVED** [review]

**Pull request merged**